web-common/src/features/sources/modal/AddStarRocksForm.svelte

```diff
@@ -0,0 +1,378 @@
+<script lang="ts">
```

Comment:
we should not create a separate StarRocks Form as we have a new abstraction to dynamically render the forms in https://github.com/rilldata/rill/pull/8467